features/admin.extensions.v1/configs/models/attribute.ts

```diff
@@ -89,4 +89,5 @@ export interface AttributeConfig {
     showCustomDialectInSCIM: boolean;
     isRowSelectable: (claim: Claim | ExternalClaim | ClaimDialect) => boolean;
     isSCIMEditable: boolean;
+    isMobileNumberVerificationByPrivilegedUsersEnabled?: boolean;
```

Comment:
Also, let's move this type of boolean configurations to deployment config as disabled features.With the unification, we are gradually getting away from using extensions.

Refer https://github.com/wso2/identity-apps/blob/master/apps/console/src/public/deployment.config.json#L589

Reply:
Changed